src/components/ImpactDialog.js

```diff
@@ -0,0 +1,208 @@
+  switch (modalType) {
+    case 'confirmImpact':
+      if (open && typeof buttonOnClick !== 'function') {
+        throw new Error('props in confirm Impact dialog are incorrect')
+      }
+      dialogContent = (
+        <>
+          <MuiDialogTitle disableTypography className={classes.root}>
+            <Typography variant="h6">
+              Are you ready to turn your Tabs into a force for good?
+            </Typography>
+          </MuiDialogTitle>
+          <MuiDialogContent>
+            <div className={classes.centerImage}>
+              <RandomGif />
+            </div>
+            <Typography className={classes.typographySpacing}>
+              Each time you open a tab, you'll be helping shelter cats get
+              adopted by{' '}
+              <a target="_blank" rel="noopener noreferrer" href="nba.com">
+                test
+              </a>
+              . Ready to get started?
+            </Typography>
+          </MuiDialogContent>
+          <MuiDialogActions>
+            <Button
+              autoFocus
+              onClick={buttonOnClick}
+              variant="contained"
+              color="primary"
+            >
+              I'M READY!
+            </Button>
+          </MuiDialogActions>
+        </>
+      )
+      break
+    case 'impactWalkMe':
+      dialogContent = (
+        <>
+          <MuiDialogTitle disableTypography className={classes.root}>
+            <Typography variant="h6">Open a new tab</Typography>
+          </MuiDialogTitle>
+          <MuiDialogContent className={classes.walkMe}>
+            <Typography id="alert-dialog-description">
+              When you do, you'll donate enough to give a shelter cat a treat
+              during training. We'll track how many treats you've given on the
+              top of the page:
+            </Typography>
+            <div className={classes.impactCounter}>
+              <ImpactCounter includeNumber number={0} progress={100} />
+            </div>
+          </MuiDialogContent>
+        </>
+      )
+      break
+    case 'claimImpactReward':
+      if (open && (!user || typeof buttonOnClick !== 'function')) {
+        throw new Error('missing props in claim impact reward dialog')
+      }
+      dialogContent = (
+        <>
+          <MuiDialogTitle disableTypography className={classes.root}>
+            <Typography variant="h6">
+              You just helped a shelter cat! Congrats!
+            </Typography>
+          </MuiDialogTitle>
+          <MuiDialogContent>
+            <div className={classes.centerImage}>
+              <RandomGif />
+            </div>
+            <Typography className={classes.typographySpacing}>
+              The more people who open tabs, the more good we can do! Get a
+              friend on board and we'll give a treat to another 10 shelter cats:
+            </Typography>
+            <div className={classes.shareContainer}>
+              <InviteFriends user={user} />
```

Comment:
Need some additional space between this and the social sharing (maybe ~8px?). May make sense to do this when you merge #124